### PR TITLE
OPENEUROPA-2127: Update components page.

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: docs/openeuropa-components.md [hunk 1]
```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
...
-            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_webtools"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_webtools/status.svg"/></a>
-            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_webtools"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_webtools.svg"/></a>
-        </td>
- <td><a href="https://github.com/imanoleguskiza">Imanol Eguskiza</a><br/>
-            <a href="https://github.com/upchuk">Daniel Sipos</a></td>
-    </tr>
-</table>
+Components are divided by track, each track has a few OpenEuropa team members responsible for it.
+
+### Editorial track
```
Comment (voidtek) [inline docs/openeuropa-components.md]: Two `##` is enough. if you want to use `###`, we should add before 
```
## Tracks

Components are divided by track, each track has a few OpenEuropa team members responsible for it.

### Editorial
```

File: docs/openeuropa-components.md [hunk 2]
```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
...
+- [OpenEuropa Editorial](https://github.com/openeuropa/oe_editorial)
+
+### Multilingual track
+
+Maintainers: [Daniel](https://github.com/upchuk), [Diogo](https://github.com/dxvargas), [Imanol](https://github.com/imanoleguskiza)
+
+Components:
+- [OpenEuropa Multilingual](https://github.com/openeuropa/oe_multilingual)
+- [OpenEuropa Translation](https://github.com/openeuropa/oe_translation)
+
```
Comment (voidtek) [inline docs/openeuropa-components.md]: epoetry and poetry client is missing.

Comment (voidtek) [reply docs/openeuropa-components.md]: oe_epoetry is missing too

File: docs/openeuropa-components.md [hunk 3]
```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
...
+### Webtools track
+
+Maintainers: [Francesco](https://github.com/brummbar), [Imanol](https://github.com/imanoleguskiza), [Sergii](https://github.com/sergepavle)
+
+Components:
+- [OpenEuropa Webtools](https://github.com/openeuropa/oe_webtools)
```
Comment (voidtek) [inline docs/openeuropa-components.md]: `oe_webtools_location` and `webtools-geocoding-provider` are missing.

File: docs/openeuropa-components.md [hunk 4]
```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
...
+- [OpenEuropa Corporate Blocks](https://github.com/openeuropa/oe_corporate_blocks)
+- [OpenEuropa RDF SKOS](https://github.com/openeuropa/rdf_skos)
+
+### Media track
+
+Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Daniel](https://github.com/upchuk)
+
+Components:
+- [OpenEuropa Media](https://github.com/openeuropa/oe_media)
+- [OpenEuropa Oembed](https://github.com/openeuropa/oe_oembed)
```
Comment (voidtek) [inline docs/openeuropa-components.md]: `media_avportal` is missing

File: docs/openeuropa-components.md [hunk 5]
```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
...
+- [OpenEuropa Search](https://github.com/openeuropa/oe_search)
+
+### Tooling track
+
+Maintainers: [Antonio](https://github.com/ademarco), [Diogo](https://github.com/dxvargas), [Victor](https://github.com/voidtek)
+
+Components:
+- [Drupal Site Template](https://github.com/openeuropa/drupal-site-template)
+- [Drupal Module Template](https://github.com/openeuropa/drupal-module-template)
+- [Task Runner](https://github.com/openeuropa/task-runner)
```
Comment (voidtek) [inline docs/openeuropa-components.md]: `oe_profile`, `composer-artifacts` and `behat-transformation-context`, `config_owner` and `drupal-core-require-dev` are missing

File: docs/openeuropa-components.md (hunk 2 above)
Comment (ademarco) [reply docs/openeuropa-components.md]: Those two are not supposed to be used, that's why we are not listing them.

File: docs/openeuropa-components.md (hunk 3 above)
Comment (ademarco) [reply docs/openeuropa-components.md]: oe_webtools_location has been moved in oe_webtools and deprecated, we could list webtools-geocoding-provider, yes although we are not the maintainers, maybe it deserves a different paragraph?

File: docs/openeuropa-components.md (hunk 4 above)
Comment (ademarco) [reply docs/openeuropa-components.md]: That's a dependency of oe_media, that's why it's not explicitly listed here

File: docs/openeuropa-components.md (hunk 5 above)
Comment (ademarco) [reply docs/openeuropa-components.md]: true, I'll add them

**Pull request merged**